content/api/commands/mount.md

```diff
@@ -5,8 +5,13 @@ componentSpecific: true
 
 <Alert type="warning">
 
-Cypress does not have a `cy.mount()` command out-of-the-box. See below for info
-on how to craft your own.
+Cypress does not have a built-in `cy.mount()` command. The command must be set
+up in your `/cypress/support/component.{js/ts}` file. By default, when you use
```

Comment:
Is there any way we can refer to this file without including the `/cypress/support/component.{js/ts}` bit? There was a lot of that in the pre-10.0 docs, which resulted in a lot of cleanup work when the default value changed. What if we called it "your component testing `supportFile`" and link that to `/guides/references/configuration#component` ?

Reply:
👍